cylc/flow/workflow_files.py

```diff
@@ -1263,15 +1263,14 @@ def infer_latest_run(
         return (path, reg)
     if not runN_path.is_symlink() or not runN_path.is_dir():
         raise WorkflowFilesError(
-            f"runN directory at {runN_path} is a broken or invalid symlink"
+            f"{runN_path} symlink not valid"
```

Comment:
It's not really a directory, and "broken" vs "invalid" seems an unnecessary distinction.